parser-typechecker/src/Unison/Builtin.hs

```diff
@@ -433,8 +438,19 @@ builtinsSrc =
     ++ hashBuiltins
     ++ fmap (uncurry B) codeBuiltins
 
+
 moveUnder :: Text -> [(Text, Type v)] -> [BuiltinDSL v]
-moveUnder prefix bs = bs >>= \(n,ty) -> [B n ty, Rename n (prefix <> "." <> n)]
+moveUnder prefix bs = bs >>= \(n,ty) -> [B n ty, Rename n (newName n)]
+  where
+    newName :: Text -> Text
+    newName name = newName' name $ RE.matchOnceText regex name
+
+    newName' _ (Just (before, _, _)) = prefix <> "." <> before
+    newName' name Nothing = prefix <> "." <> name
+    r :: String
+    r = "\\.v[0-9]+"
+    regex :: RE.Regex
+    regex = RE.makeRegexOpts (RE.defaultCompOpt { RE.caseSensitive = False }) RE.defaultExecOpt r
```

Comment:
I think stripping out the `v2` doesn't belong in this function. I'd keep this function the same as it was, and then add a `stripVersionNumber` function that goes from `[BuiltinDSL v] -> [BuiltinDSL v]`.